From 6d235f2cd835d5d3e00dee43ed29bdf2c5ca80d5 Mon Sep 17 00:00:00 2001
From: Ralf Habacker <ralf.habacker@freenet.de>
Date: Tue, 23 May 2017 19:52:20 +0200
Subject: [PATCH 240/258] Fix 'duplicated symbol' compile error on Windows.

Because the whole project uses cmake automoc support additional
calls to automoc4 causes 'duplicated symbol' errors on Windows.
---
 kmymoney/plugins/onlinetasks/national/CMakeLists.txt | 1 -
 kmymoney/plugins/onlinetasks/sepa/CMakeLists.txt     | 1 -
 2 files changed, 2 deletions(-)

diff --git a/kmymoney/plugins/onlinetasks/national/CMakeLists.txt b/kmymoney/plugins/onlinetasks/national/CMakeLists.txt
index d3e7c44..5710c70 100644
--- a/kmymoney/plugins/onlinetasks/national/CMakeLists.txt
+++ b/kmymoney/plugins/onlinetasks/national/CMakeLists.txt
@@ -20,7 +20,6 @@ kde4_add_ui_files( konlinetasks_national_SCRS
 )
 
 # Make this plugin unit testable
-automoc4(konlinetasks_national_OBJECTS konlinetasks_national_SCRS)
 add_library( konlinetasks_national_OBJECTS OBJECT ${konlinetasks_national_SCRS} )
 
 # Add plugin which is not allowed in the unit test
diff --git a/kmymoney/plugins/onlinetasks/sepa/CMakeLists.txt b/kmymoney/plugins/onlinetasks/sepa/CMakeLists.txt
index 3c2d9db..8415c69 100644
--- a/kmymoney/plugins/onlinetasks/sepa/CMakeLists.txt
+++ b/kmymoney/plugins/onlinetasks/sepa/CMakeLists.txt
@@ -12,7 +12,6 @@ kde4_add_ui_files( sepaOnlineTasks_SRCS
   ui/sepacredittransferedit.ui
 )
 
-automoc4( konlinetasks_sepa_OBJECTS sepaOnlineTasks_SRCS)
 add_library( konlinetasks_sepa_OBJECTS OBJECT ${sepaOnlineTasks_SRCS} )
 
 # Add actual plugin here it is not part of the objects
-- 
2.9.4

